.github/workflows/cifuzz.yml

```
@@ -0,0 +1,26 @@
+name: CIFuzz
+on: [pull_request]
```

Not part of review: I'm curious, looking at the actions below, how do `build_fuzzers` and `run_fuzzers` retrieve the `cilium/cilium` code from the PR? I have quickly looked at the Dockerfiles / documentation but didn't find the answer immediately, perhaps you can point me in the right direction?

---

Am not sure about the specific details, but I think you're looking at this function here in terms of downloading the specific commit/pr: https://github.com/google/oss-fuzz/blob/master/infra/cifuzz/continuous_integration.py#L184-L195

Which Dockerfile are you referring to? This one https://github.com/google/oss-fuzz/blob/master/projects/cilium/Dockerfile ?

I don't think it will set a GitHub commit status (am not entirely sure what this is) but simply state whether the action was successful or not. If a crash is found the action will fail and you can get the triggering test as described here https://google.github.io/oss-fuzz/getting-started/continuous-integration/#understanding-results

---

>Am not sure about the specific details, but I think you're looking at this function here in terms of downloading the specific commit/pr: https://github.com/google/oss-fuzz/blob/master/infra/cifuzz/continuous_integration.py#L184-L195

+1

>I don't think it will set a GitHub commit status (am not entirely sure what this is) but simply state whether the action was successful or not. If a crash is found the action will fail and you can get the triggering test as described here https://google.github.io/oss-fuzz/getting-started/continuous-integration/#understanding-results

I also don't know what this is.
In case of any failure, the CI will fail with a red X.
If the failure is because of a crash, you will be able to download the crashing testcase.

---

OK so it does add a check to the PR checks if it shows up with a failure in the Actions tab.

---

Can you please add an ignore for `Documentation` to the `pull_request` trigger, so that we don't run fuzzing for documentation-only PRs?

```suggestion
on:
  pull_request:
    paths-ignore:
      - 'Documentation/**'
```